README.md

```diff
@@ -4,26 +4,37 @@
 
 ## Running
 
-The Explorer can be run by using `docker-compose`:
+The Explorer can be run by using `docker-compose`. This will build the docker images and run multiple instances of the explorer for mainnet and all supported testnets.
+
 ```sh
 docker-compose build
 docker-compose up -d
 ```
 
-It's important to update the [frontend config files](https://github.com/raiden-network/explorer/tree/master/frontend/src/assets/config) because otherwise the official Explorer backend is used.
-
 ## Frontend
 
-The frontend is built using [Angular](https://angular.io) and required NodeJS. It can be found in the `frontend` directory.
+The frontend is built using [Angular](https://angular.io) and requires NodeJS. It can be found in the `frontend` directory.
 
-To launch a development version of the frontend, install angular globally and run the angular development server:
-```bash
-npm install -g @angular/cli
-cd angular-app
+To launch a development version of the frontend, install the dependencies and run the angular development server:
+
+```sh
+cd frontend
 npm install
-ng serve
+npm start
 ```
 
+It's important to update the [frontend config files](https://github.com/raiden-network/explorer/tree/master/frontend/src/assets/config) because otherwise the official Explorer backend is used.
```

Comment:
So what would I need to do in exact to connect it to the backend as described below?

Reply:
In `config.development.json` you need to set `backend_url` to your backend, by default `http://localhost:4567/json`.